### PR TITLE
[3508] Add additional padding to InputField

### PR DESCRIPTION
**Pull request by TGazica**
### 🎯 Goal

Add additional padding to `InputField`.

### 🛠 Implementation details

Split `innerPadding` of the `InputField` to `verticalPadding` and `horizontalPadding`.

### 🎨 UI Changes

| Before | After |
| --- | --- |
| Screenshot_20220525_163311 | Screenshot_20220525_163437 |

### 🧪 Testing

1. Run on develop and check the padding
2. Run on this branch and check the padding

#### General
- [x] Assigned a person / code owner group (required)
- [ ] Thread with the PR link started in a respective Slack channel (#android-chat-core or #android-chat-ui) (required)
- [x] PR targets the `develop` branch
- [x] PR is linked to the GitHub issue it resolves

#### Code & documentation
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (KDocs, docusaurus, tutorial)

### ☑️Reviewer Checklist
- [ ] UI Components sample runs & works
- [ ] Compose sample runs & works
- [ ] UI Changes correct (before & after images)
- [ ] Bugs validated (bugfixes)
- [ ] New feature tested and works
- [ ] Release notes and docs clearly describe changes
- [ ] All code we touched has new or updated KDocs

### 🎉 GIF

giphy

### CODE REVIEW

File: stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/composer/InputField.kt [hunk 1]
```diff
@@ -56,6 +56,24 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
...
+            "onValueChange: (String) -> Unit," +
+            "modifier: Modifier = Modifier," +
+            "enabled: Boolean = true," +
+            "maxLines: Int = Int.MAX_VALUE," +
+            "border: BorderStroke = BorderStroke(1.dp, ChatTheme.colors.borders)," +
+            "horizontalPadding: Dp = 18.dp," +
+            "verticalPadding: Dp = 8.dp," +
+            "decorationBox: @Composable (innerTextField: @Composable () -> Unit) -> Unit)",
+        imports = ["io.getstream.chat.android.compose.ui.components.composer"]
+    ),
```
Comment (bychkovdmitry) [inline stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/composer/InputField.kt:74]: To be honest I have never understood how this is supposed to work. Doesn't work for me when autocorrecting the warning in Android Studio. Could you check if the following snippet: 

```
@Deprecated(
    message = "Deprecated in favor of new InputField implementation providing more padding customization options.",
    replaceWith = ReplaceWith(
        expression = "InputField(value, onValueChange, modifier, enabled, maxLines, border, horizontalPadding, verticalPadding, decorationBox)",
        imports = ["io.getstream.chat.android.compose.ui.components.composer"]
    ),
    level = DeprecationLevel.WARNING
)
```
Screenshot 2022-05-25 at 18 04 34

Comment (TGazica) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/composer/InputField.kt:74]: Yes it works like this but is pretty broken and will add all the missing parameters not passed in. I honestly don't think this change is worth it, but I can add it.

Comment (filbabic) [reply stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/composer/InputField.kt:74]: Do we really need to deprecate the old input field? We've just exposed one new parameter that _has a default value_ hence it's not going to be breaking for any of our users, they can just expand on their behavior.

**Review by bychkovdmitry — APPROVED** [review]
LGTM

**Pull request merged**